assignments/array-methods.js

```diff
@@ -52,25 +52,29 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":48,"first_name":"Anderea","last_name":"MacGiolla Pheadair","email":"amacgiollapheadair1b@xing.com","shirt_size":"2XL","company_name":"Kwimbee","donation":214},
 {"id":49,"first_name":"Bel","last_name":"Alway","email":"balway1c@ow.ly","shirt_size":"S","company_name":"Voolia","donation":107},
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
-
+// Initial commit (no change)
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(runner => fullName.push(`${runner.first_name} ${runner.last_name}`));
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+runners.map(runner => allCaps.push(runner.first_name.toUpperCase()));
```

Comment:
Nice work! You can also set it up like:
```suggestion
let allCaps = runners.map(runner => runner.first_name.toUpperCase());
```
This is saying let the array that maps creates be called allCaps. To populate that array, map over each runner and return the runner's capitalized first name. The array will then contain a capitalized first name for each runner.